src/main/java/io/trino/plugin/tiledb/TileDBSplitManager.java

```diff
@@ -275,18 +279,37 @@ else if (leftOvers > 0) {
                     high += 1;
                     leftOvers--;
                 }
-
                 // Only set the range if the values are not equal or if the low
                 // and high are the bounds must also be the same
                 if (low > high) {
-                    LOG.warn("Low > high while setting ranges."); //TODO investigate why
+                    LOG.warn("Low > high while setting ranges.");
                     return ranges;
                 }
-                else if (low == high) {
-                    lowerInclusive = true;
+                if (low == high) {
```

Comment:
Can we simplify this? `lowerInclusive = true` and `highInclusive = true` is the default case above (line 261), so it seems like this new section might be redundant with the check above on line 264.

Reply:
I refactored it. Minimal changes were needed after all 🙃

Reply:
Awesome, nice work.